### DISCUSSION TITLE
Coil3 vs Coil2

### DISCUSSION THREAD

**Discussion by T8RIN** [coil-kt/coil#2672] [Q&A] [answered]
I don't really need multiplatform support, and i have huge amount of things which rely on coil, so is there some performance benefits over coil2 ? I want to stay up to date, but rewriting the entire app and few libraries is sounds awful

**Comment by colinrtwhite** [comment, marked as answer]
Coil 3 does have performance improvements, API improvements, and bug fixes that aren't present in Coil 2. If Coil 2 is working well for you then you don't have to update, but Coil 2's support will be limited and it is not going to get new features or minor bug fixes going forward. If your goal is to stay up to date I highly recommend updating.
